gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/converter/HiveAvroToOrcConverter.java

```diff
@@ -76,8 +72,8 @@ public Schema convertSchema(Schema inputSchema, WorkUnitState workUnit) throws S
     // TODO: Add num of buckets (from config)
 
     // Avro table name and location
-    String avroTableName = conversionEntity.getHiveUnit().getTableName();
-    String avroTableLocation = conversionEntity.getHiveUnit().getLocation().get();
+    String avroTableName = conversionEntity.getHiveTable().getTableName();
+    String avroTableLocation = conversionEntity.getHiveTable().getSd().getLocation();
```

Comment:
In case that converter is running for a partition, the location will be wrong (of table). We should check if this conversion entity has a partition, if so use its location 
Also, the avroTableLocation can be changed to avroDataLocation or similar, since it can be table or partition location based on what the conversion is happening